src/rules/handlers.ts

```diff
@@ -209,16 +209,24 @@ export class CallbackHandlerData extends Serializable {
 export class PassThroughHandlerData extends Serializable {
     readonly type: 'passthrough' = 'passthrough';
 
+    constructor(private forwardToLocation?: string) {
+        super();
+    }
+
     buildHandler() {
         return _.assign(async (clientReq: OngoingRequest, clientRes: express.Response) => {
             const { method, originalUrl, headers } = clientReq;
             let { protocol, hostname, port, path } = url.parse(originalUrl);
+            if (this.forwardToLocation) {
+                ({ protocol, hostname, port } = url.parse(this.forwardToLocation));
```

Comment:
So we don't use the `forwardToLocation` path at all, right?

I think either:

Reply:
Good idea. Now throwing in this case.